using-mobile-extensions/adobe-edge-consent/README.md

```diff
@@ -0,0 +1,162 @@
+# Adobe Experience Platform Edge Consent
+
+## Configure the Adobe Experience Platform Edge Consent extension in Experience Platform Launch
+
+1. In Experience Platform Launch, in your mobile property, click the **Extensions** tab.
+2. On the **Catalog** tab, locate or search for the **Consent Collection** extension, and click **Install**.
+3. Set your desired default consent level.
+4. Click **Save**.
+5. Follow the publishing process to update SDK configuration.
+
```

Comment:
Do we want to add a note that in Launch, Edge and Edge Identity should be installed as well?  In the examples below we specify to import Edge and Identity but don't state that those launch extensions should be installed.  I'm writing the Edge Identity readme and wondering about the same. Thoughts?

Reply:
We can add a comment right below the title:
```
The Adobe Experience Platform Consent Collection mobile extension enables consent preferences collection from your mobile app when using the Adobe Experience Platform Mobile SDK <link to core> and the Edge Network extension <link to Edge>
```

And then in the Edge extension we should mention Edge Identity extension